sections/installation.js

```diff
@@ -102,6 +102,26 @@ export default [
     `
   },
   {
+    type: "info",
+    content: "Note: The database version can be added in knex configuration, when you use the PostgreSQL adapter to connect a non-standard database."
+  },
+  {
+    type: "code",
+    language: "js",
+    content: `
+      var knex = require('knex')({
+        client: 'pg',
+        version: 'v7.2.1-0',
```

Comment:
I think this is wrong... it should be something like e.g. `9.5` to be able to select which postgres features will be available. Now this example looks like you are giving the string that is parsed from the version string provided by custom database.